OtripleS.Web.Api/Properties/launchSettings.json

```diff
@@ -1,11 +1,11 @@
-﻿{
+{
```

Comment:
This is specific to your local settings, do not check-in this.

Reply:
Can u explain me this.

Reply:
Revert the changes in this file.

Reply:
Don't know how i messed up this.. I tried to fix